src/client/js/chart-options.jsx

```diff
@@ -83,6 +89,9 @@ class ChartOptions extends React.Component {
                   onChange={this.handleStartDateTimeChange.bind(this)}
                   InputLabelProps={{ shrink: true }} />
               </FormControl>
+			  <Tooltip title="Enter the date and time you would like the sample to start.">
```

Comment:
Same here. Fix this indentation. Uses spaces only, no tabs.

Reply:
Should be correct now, still a little confused on proper formatting.